api/cluster/templater.go

```diff
@@ -160,6 +161,19 @@ func createPredictorSpec(modelService *models.Service, config *config.Deployment
 					Container: corev1.Container{
 						Name:      kserveconstant.InferenceServiceContainerName,
 						Resources: resources,
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								TCPSocket: &corev1.TCPSocketAction{
+									Port: intstr.IntOrString{
```

Comment:
Why do wr use port 9000 for tensorflow? This is grpc port and not guaranteed to be open. Can we make use endpoiny in port 8080?

Reply:
I didnt know that this value can be chosen by me. Initially I thought it is server configured, so I copied this value from someone else online. Was still experimenting in the last commit. Willl update this as well.

Reply:
If you are experimenting, I suggest to do it in the dev cluster by creating the inference service manifest manually so that you can observe if it's correct/not right away and iterate faster. If you use CI to experiment it will be very slow and wasted your time.